tests/sql/setup_test_docx_etl.sql

```diff
@@ -0,0 +1,18 @@
+insert into "Bills" (legislative_id, legislative_session, bill_number, session_type, chamber)
```

Comment:
consider making this idempotent, using something like `insert ignore` command. As it stands, if in any situation the .sql query is run more than once, an error will be thrown

Reply:
Postgres SQL doesn't have an insert ignore command, but it does have something similar ("On conflict (value) do nothing"), so I've used that instead.